Try_Catch.cpp

```diff
@@ -0,0 +1,73 @@
+#include <iostream>
+#include <stdexcept>
+
+using namespace std;
+
+class CarDamageException : public exception {
+private:
+    string message;
+public:
+    CarDamageException(const char* msg) : message(msg) {}
+    const char* what() const noexcept override {
+        return message.c_str();
+    }
+};
+
+class BrakeFailureException : public exception {
+private:
+    string message;
+public:
+    BrakeFailureException(const char* msg) : message(msg) {}
+    const char* what() const noexcept override {
+        return message.c_str();
+    }
+};
+
+class Car {
+public:
+    void driveSafely(bool collision, bool brakeFailure) {
+        if (collision) {
+            throw CarDamageException("Car damaged due to collision.");
+        } else if (brakeFailure) {
+            throw BrakeFailureException("Brake Failure Occurred");
+        }
+    }
+};
+
+class AccidentHandler {

```

Comment:
Can have a better name for this class instead of **AccidentHandler**

Reply:
Done